hpx/hpx_main.hpp

```diff
@@ -9,6 +10,18 @@
 #include <hpx/config.hpp>
 #include <hpx/hpx_init.hpp>
 
+// We support different implementation depending upon the Operating
+// System in use.
+#if defined(__linux__)
```

Comment:
We usually check for Linux as follows:
```
#if defined(__linux) || defined(__linux__) || defined(linux)
...
#endif
```

Different flavors of Linux define different pp constants.

Reply:
I will update it as you have described.